### ISSUE TITLE
I'd like op hacks for servers with no anticheat

### ISSUE THREAD

**Issue by AxenKing** [ImpactDevelopment/ImpactIssues#3186]
I would like some hacks like 'God mode' , 'Invsee' and other, for servers with no anticheat as my friends are making a server with no anticheat. This may sound like insane features but I just want it for purely the purpose of single payer and no anticheat servers.

**Comment by marconovino** [comment]
yeah bro i also love impossible features, @brady when are we getting the item spawner 9000???!!!

**Comment by AcaiBerii** [comment]
> yeah bro i also love impossible features, @brady when are we getting the item spawner 9000???!!!

what about the .give command?!??!

**Comment by Doogie13** [comment]
I was told these issues were bad but jesus christ. Are you old enough to play minecraft yet?

**Comment by Zennara** [comment]
> I was told these issues were bad but jesus christ. Are you old enough to play minecraft yet?

.invsee is a currently available cheat. It will show a players hotbar and inventory. I believe its available on Liquidbounce.
